### PR TITLE
feat(lambda): make node version configurable

### PR DESCRIPTION
**Pull request by herschel666**

### CODE REVIEW

File: packages/lambda/package.json [hunk 1]
```diff
@@ -32,5 +31,8 @@
     "serverless-http": "^2.3.0",
     "strip-indent": "^3.0.0"
   },
+  "devDependencies": {
+    "aws-sdk": "^2.546.0"
```
Comment (ZauberNerd) [inline packages/lambda/package.json]: How is this supposed to work? The deployment `hops lambda deploy` needs the `aws-sdk` and it won't be installed when it is a peer dependency.

Comment (herschel666) [reply packages/lambda/package.json]: My bad. Thought this was ending up in the bundle, which is not the case. Will remove that commit.

File: package.json [hunk 2]
```diff
@@ -49,7 +49,7 @@
       },
       {
         "repository": "https://github.com/foosbyte/kicker-trainer.git",
-        "branch": "next",
+        "branch": "update-node-and-hops",
```
Comment (jhiode) [inline package.json]: Ah wait, we have to remove this first 😬

Comment (herschel666) [reply package.json]: I actually thought about merging it with this change, then merging the kicker-trainer (while keeping the branch) & then revert this change in another PR.

**Pull request merged**